2-Draft-Accepted/RFCNNNN-Support-Experimental-Features.md

```diff
@@ -327,8 +337,11 @@ For experimental features in PowerShell engine,
 an internal enum `EngineExperimentalFeatures` will be used to track the names of all available engine experimental features.
```

Comment:
internal? Should this be public rather than internal?

Reply:
The idea is to add a cmdlet to expose this information to users.
The members of the enum will change consistently, so it would be a consistent breaking change if we make it public.

Reply:
Consider the possibility of third party experiments relying on engine experiments.  For example, an editor experiment that depends on a language experiment.  The editor experiment would need to either define it's own then check for the language experiment being enabled or expose it's capabilities as the same experiment.

Scripts and modules may also take dependencies on engine experiments so, whatever the form, the cooperation is likely desirable; especially for new features and capabilities.

Reply:
Updated the PR to use a read-only collection to track all engine experimental features.

For the scenario that third party experiments depending on engine experiments, it's covered in the `Experimental Feature Dependency` section. A third party experiment can specify a dependency on an engine experimental feature, but the third party experiment needs to be a separate experimental feature.